test/verify/test_isnan.cpp

```diff
@@ -0,0 +1,21 @@
+#include <limits>
+#include "verify_program.hpp"
+#include <migraphx/program.hpp>
+#include <migraphx/generate.hpp>
+#include <migraphx/make_op.hpp>
+
+struct test_abs : verify_program<test_abs>
+{
+    migraphx::program create_program() const
+    {
+        migraphx::program p;
+        auto* mm = p.get_main_module();
+        auto x   = mm->add_parameter("x", migraphx::shape{migraphx::shape::float_type, {2, 3}});
```

Comment:
I think you can drop the second dimension and just do `migraphx::shape{migraphx::shape::float_type, {2}}`